tensorflow_datasets/image/__init__.py

```diff
@@ -42,10 +42,7 @@
 from tensorflow_datasets.image.open_images import OpenImagesV4
 from tensorflow_datasets.image.quickdraw import QuickdrawBitmap
 from tensorflow_datasets.image.rock_paper_scissors import RockPaperScissors
-<<<<<<< HEAD
 from tensorflow_datasets.image.stanford_dogs import StanfordDogs
```

Comment:
Arrange the import names alphabetically :)

Reply:
Done!